presto-main/src/main/java/io/prestosql/execution/QueryManagerConfig.java

```diff
@@ -295,6 +297,18 @@ public QueryManagerConfig setQueryMaxCpuTime(Duration queryMaxCpuTime)
         return this;
     }
 
+    public DataSize getQueryMaxScanPhysicalBytes()
```

Comment:
Add `@NotNull` and return `Optional<DataSize>`

Reply:
ok

Reply:
Done